js/views/storyeditview/storyeditview.js

```diff
@@ -261,15 +261,15 @@ var StoryEditView = Marionette.CompositeView.extend(
   },
 
   addPassageEndGameGroupSuccessNumberResult: function(name, left, top) {
-    this.addPassage(name, left, top, 'addPassageEndGameGroupSuccessNumberResult');
+    this.addPassage(name, left, top, 'passageEndGameGroupSuccessNumberResult');

```

Comment:
Hahaha yeah. Those class names are really long, and long to type. Definitely open to changing them. 

My reasoning for such a long name--even though currently, the `passageEndGameGroupSuccessNumberResult` is the only type of group endgame passage, I'm anticipating the possibility of us adding more group endgame passage types. 

The appended "result" was to make it clear that the passage indicated an endgame result. In retrospect, that's probably unnecessary. 

However, it might seem unlikely that we add new classes, in which case we should totally shorten it.